kirby/component/markdown.php

```diff
@@ -34,9 +35,10 @@ public function defaults() {
    * transforms the given markdown to HTML
    * 
    * @param string $markdown
+   * @param Field $field
    * @return string
    */
-  public function parse($markdown) {
+  public function parse($markdown, Field $field = null) {
```

Comment:
Why is this param optional here?

If for backward compatibility: I think we should make it required to avoid bugs when the param isn't passed for some reason (custom components wouldn't work but we wouldn't notice it).